content/coronavirus_education_page.yml

```diff
@@ -69,6 +69,8 @@ content:
               url: /government/publications/coronavirus-covid-19-maintaining-further-education-provision
             - label: Safe working in education, childcare and children’s social care
               url: /government/publications/safe-working-in-education-childcare-and-childrens-social-care
+            - label: Supporting children and young people with SEND as schools and colleges prepare for wider opening
+              url: /government/publications/coronavirus-covid-19-send-risk-assessment-guidance
```

Comment:
Should the link point at the HTML document rather than it's parent? That is should this line be:
```yaml
url: /government/publications/coronavirus-covid-19-send-risk-assessment-guidance/coronavirus-covid-19-send-risk-assessment-guidance
```

Reply:
I think linking to the parent page is fine. I asked on slack but didn't get a reply, so I checked other pages on the landing page and they link to the parent pages too.